requirements/edx/base.in

```diff
@@ -39,7 +39,7 @@ defusedxml
 Django<1.12                     # Web application framework
 django-babel-underscore             # underscore template extractor for django-babel (internationalization utilities)
 django-config-models>=1.0.0         # Configuration models for Django allowing config management with auditing
-django-cors-headers==2.1.0          # Used to allow to configure CORS headers for cross-domain requests
+django-cors-headers==3.0.2          # Used to allow to configure CORS headers for cross-domain requests
```

Comment:
It looks like one of the [changes in 3.0.0](https://github.com/adamchainz/django-cors-headers/blob/master/HISTORY.rst#300-2019-05-10) requires some setting changes both here and in some private configuration repositories.  This is a setting that doesn't really impact the test runs, but can cause problems in stage and production if set incorrectly.  Probably should stick with 2.5.3 for now, and I'll create another ticket for someone here to look into coordinating an update of that setting with a further upgrade later.

In case you haven't seen it, there's a wiki page with links to changelogs for many of the packages we use in Open edX - https://openedx.atlassian.net/wiki/spaces/TE/pages/1001521320/Python+Package+Changelogs .  It can be useful in reviewing recent changes in outdated packages to see if there are any breaking changes like this that we need to account for.